wiki/Home.md

```diff
@@ -0,0 +1,26 @@
+
+Welcome to the unified-codes wiki!
+
+"unified-codes" is intended to complately replace the original universal drug codes server. 
+* The "original" universal drug codes server is currently located at https://sussol.github.io/universal-drug-codes-server/.
+* "unified-codes" (aka V2) is not yet deployed.
+
+## System Overview/Design [TO COMPLETE]
+<!-- This is quite high level - A diagram may be useful -->
+* Database 
+  * [Dgraph](https://dgraph.io/docs/)
+* APIs 
+  * GraphQL served by [Apollo](https://www.apollographql.com/docs)
+  * REST served by [Fastify](https://www.fastify.io/)
+* Web UI 
+  * [React](https://reactjs.org/docs)
+* Authentication
+  * TBC
```

Comment:
Integration with an open ID compliant server, using JSON web tokens. Currently implemented using [Keycloak](https://www.keycloak.org/) as the authentication server ( see the data-service sub-folder for the 'get-you-up-and-running' docker implementation )

Reply:
Oh nice, thanks for the snippet. Added it to the TBC section for now while finding a home for these details